Dropdown/Dropdown.module.less

```diff
@@ -99,60 +90,130 @@
-					font-weight: @agate-dropdown-item-font-weight;
-					line-height: @agate-dropdown-item-line-height;
-
-					&.radioItem {
-						border-color: @agate-dropdown-item-border-color;
-						border-width: @agate-dropdown-item-border-width;
-						padding-left: @agate-dropdown-item-padding;
-						padding-right: @agate-dropdown-item-padding;
-
-						.focus({
-							background-color: @agate-dropdown-item-focus-bg-color;
-						});
+			&.open {
+				.button {
+					.bg {
+						border-radius: @agate-dropdown-button-open-border-radius;
+						border-bottom-color: @agate-dropdown-button-open-border-color;
+						transition: none;
 					}
+
+					.focus({
+						.bg {
+							border-bottom-color: @agate-dropdown-button-open-border-color;
+						}
+					});
 				}
 			}
 
 			&.upDropdownButton {
-				.button {
-					&.open {
+				&.open {
+					.button {
 						.bg {
 							border-radius: @agate-dropdown-up-button-open-border-radius;
 							border-bottom-color: @agate-dropdown-up-button-open-border-bottom-color;
 							border-top-color: @agate-dropdown-button-open-border-color;
 						}
+
+						.focus({
+							.bg {
+								border-bottom-color: @agate-dropdown-up-button-open-border-bottom-color;
+							}
+						})
 					}
 				}
+			}
+		}
+	});
+}
 
-				.dropdownList {
-					border-radius: @agate-dropdown-up-button-list-border-radius;
-				}
+.dropdownList {
+	.applySkins({
+		border-bottom: @agate-dropdown-border-width @agate-dropdown-border-style @agate-dropdown-border-color;
+		border-radius: @agate-dropdown-list-border-radius;
+		background-color: @agate-dropdown-list-bg-color;
+		overflow: hidden;
+
+		.item {
+			border-color: @agate-item-border-color;
+			border-width: @agate-dropdown-item-border-width;
+
+			.focus({
+				border-color: @agate-dropdown-item-focus-border-color;
+			});
+
+			&.selected {
+				color: @agate-item-color;
 			}
+		}
 
-			.dropdownListWithScroller {
-				.scroller {
+		.dropDownListItem {
+			font-size: @agate-dropdown-item-font-size;
+			font-weight: @agate-dropdown-item-font-weight;
+			line-height: @agate-dropdown-item-line-height;
+
+			&.radioItem {
+				border-color: @agate-dropdown-item-border-color;
+				border-width: @agate-dropdown-item-border-width;
+				padding-left: @agate-dropdown-item-padding;
+				padding-right: @agate-dropdown-item-padding;
+
+				.focus({
+					background-color: @agate-dropdown-item-focus-bg-color;
+				});
+			}
+		}
+
+		&.smallest {
+			width: @agate-dropdown-list-smallest-width;
+		}
+
+		&.small {
+			width: @agate-dropdown-list-small-width;
+		}
+
+		&.medium {
+			width: @agate-dropdown-list-medium-width;
+		}
+
+		&.large {
+			width: @agate-dropdown-list-large-width;
+		}
+
+		&.x-large {
+			width: @agate-dropdown-list-x-large-width
+		}
+
+		&.huge {
+			width: @agate-dropdown-list-huge-width;
+		}
+
+		&.above {
+			border-radius: @agate-dropdown-up-button-list-border-radius;
+		}
+
+		&.dropdownListWithScroller {
+			height: @agate-dropdown-list-height;
```

Comment:
Could this be replaced with `max-height` without this class?

Reply:
mx-height does not activate the Scrollbar. I moved height to .dropdownList